core/src/labels/spriteLabel.cpp

```diff
@@ -36,59 +35,54 @@ void SpriteLabel::applyAnchor(const glm::vec2& _dimension, const glm::vec2& _ori
     // Transform anchor direction from anchor space (centered)
     // to local sprite space (lower-left corner for the sprite)
     m_anchor = direction * glm::vec2(-0.5, 0.5) + glm::vec2(0.5);
-}
 
-glm::vec2 SpriteLabel::anchor() const {
-    glm::vec2 anchor;
-    anchor.x = -(m_dim.x * m_anchor.x);
-    anchor.y =  (m_dim.y * m_anchor.y);
-    return anchor;
+    m_anchor.x = -(m_dim.x * m_anchor.x);
+    m_anchor.y =  (m_dim.y * m_anchor.y);
 }
 
 void SpriteLabel::updateBBoxes(float _zoomFract) {
     glm::vec2 halfSize = m_dim * 0.5f;
     glm::vec2 sp = m_transform.state.screenPos;
     glm::vec2 dim = m_dim + glm::vec2(m_extrudeScale * 2.f * _zoomFract);
-    m_obb = OBB(sp.x + halfSize.x, sp.y - halfSize.y, m_transform.state.rotation, dim.x, dim.y);
-    m_aabb = m_obb.getExtent();
-}
 
-void SpriteLabel::align(glm::vec2& _screenPosition, const glm::vec2& _ap1, const glm::vec2& _ap2) {
+    if (m_occludedLastFrame) { dim += 2; }
```

Comment:
I see, it's a good idea but I think this value should also be scaled by the pixel scale.